api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessor.java

```diff
@@ -65,7 +69,10 @@ public MessageStatus processMessage(Message message) {
 
         paymentCycleService.updatePaymentCycle(currentPaymentCycle, decision);
 
-        if (decision.getEligibilityStatus() == ELIGIBLE) {
+        if (decision.getVoucherEntitlement().getTotalVoucherValueInPence() == 0) {
```

Comment:
I also don't think this is right. Having tested this scenario, if there are no vouchers, then the decision that comes back from the eligibility check is INELIGIBLE, so I don't think this will work.

Reply:
Why would the decision being INELIGIBLE stop this check working?